lib/util.js

```diff
@@ -38,7 +38,7 @@ const getCommands = config => {
       `pbcopy < ${config.rsa_filename}.pub`
     ];
   } else {
-    [
+    return [
```

Comment:
Fucking return was missing causing cli to fail in both windows and linux systems.